File: apps/app-template/android/arsenal/libraries/library-base/src/main/java/com/smart/library/bundle/model/CXHybirdModuleBundleModel.kt
package com.smart.library.bundle.model

data class CXHybirdModuleBundleModel(
    var moduleName: String,
    var moduleConfigList: MutableList<CXHybirdModuleConfigModel> = mutableListOf(),
    var moduleNextConfig: CXHybirdModuleConfigModel? = null
)

File: apps/app-template/ios/arsenal/libraries/library-base/src/bundle/model/CXHybirdModuleBundleModel.swift
import Foundation

public class CXHybirdModuleBundleModel: NSObject {
    public var moduleName: String = ""
    public var moduleConfigList: MutableList<CXHybirdModuleConfigModel> = MutableList<CXHybirdModuleConfigModel>()
    public var moduleNextConfig: CXHybirdModuleConfigModel? = nil

    public init(_ name: String, _ list: MutableList<CXHybirdModuleConfigModel>) {
        self.moduleName = name
        self.moduleConfigList = list
    }

    public init(_ name: String, _ moduleNextConfig: CXHybirdModuleConfigModel?) {
        self.moduleName = name
        self.moduleNextConfig = moduleNextConfig
    }


    public override var description: String {
        return "(name:\(self.moduleName) configSize:\(self.moduleConfigList.size)) nextConfig:\(moduleNextConfig?.moduleName ?? "nil")"
    }

}
